docs/react/getting-started.en-US.md

```diff
@@ -0,0 +1,174 @@
+---
+order: 1
+title: Getting Started
+---
+
+Ant Design React is dedicated to providing a **good development experience** for programmers.
+
+---
+
+You had better learn [React](http://facebook.github.io/react/) and [ES2015](http://babeljs.io/docs/learn-es2015/) first.
```

Comment:
Before delving into Ant Design React, a good knowledge of [React](http://facebook.github.io/react/) and [JavaScript ES2015](http://babeljs.io/docs/learn-es2015/) is needed.